src/App.js

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import './App.css';
-import SearchBar from './Components/SearchBar';
+import Home from './Components/SearchBar';
```

Comment:
You should import Home from Home instead of SearchBar
```
import Home from './Components/Home'; 
import { PATHS } from './config/routes';
```

Reply:
ESLint tells me that "'PATHS' is declared but its value is never read."